libs/ml/examples/cbow_simple/dataloader.hpp

```diff
@@ -0,0 +1,37 @@
+#pragma once
```

Comment:
I think this is an exact copy of our abstract definition of a dataloader: ml/include/ml/dataloaders/dataloader.hpp
if so, please delete this and inherit from that one